app-text/podofo/files/0001-FIX-ColorTest-Fixed-test-under-linux-when-compiled-w.patch

```diff
@@ -0,0 +1,35 @@
+From da91101b7afafa958669e11d0cabaeee3ff50b39 Mon Sep 17 00:00:00 2001
```

Comment:
please link to the upstream commit

Reply:
Sure, it's upstream commit da91101b7afafa958669e11d0cabaeee3ff50b39 :P